medusa/clients/torrent/generic.py

```diff
@@ -133,6 +97,12 @@ def _add_torrent_file(self, result):
         """
         raise NotImplementedError
 
+    def _check_path(self):
+        """Check if the destination path is correct."""
+        log.debug('dummy check path function')
```

Comment:
I don't think the log adds value here

Reply:
That's a matter of opinion, I needed it during testing and debugging, but feel free to remove it.